Makefile

```diff
@@ -110,6 +153,8 @@ quickdist: .distclean plugin.json
 
 dist: vendor .npminstall quickdist
 	@echo Building plugin
+	@echo Remove timezones from plugin.json
+	$(call UpdatePluginFile)
```

Comment:
Why is this being called in `dist` but `GetTimeZone` is called from `quickdist`? I think both should be called from `quickdist`.